File: dev/requirements/screen/cases/laugh-strip-dim.3.7.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 3.7 — the room quiet enough to speak into — the same strip dim.
let laughStripDim_3_7 = RequirementCase(id: "3.7") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "laugh-strip-dim", id: "3.7")
    }
}
#endif

File: dev/requirements/screen/cases/laugh-strip-dim.3.7.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 3.7 — Below the threshold the same strip is dim.
final laughStripDim_3_7 = RequirementCase('3.7', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.laughStrip,
    slug: 'laugh-strip-dim',
    id: '3.7',
  );
});
